java/statestorev2/src/main/java/sleeper/statestorev2/StateStoreArrowUploadFileStore.java

```diff
@@ -28,35 +28,31 @@
 import sleeper.core.partition.Partition;
 import sleeper.core.properties.instance.InstanceProperties;
 import sleeper.core.properties.table.TableProperties;
-import sleeper.core.statestore.transactionlog.snapshot.TransactionLogSnapshot;
 import sleeper.core.statestore.transactionlog.state.StateStoreFiles;
-import sleeper.core.statestore.transactionlog.state.StateStorePartitions;
 import sleeper.statestorev2.transactionlog.snapshots.TransactionLogSnapshotMetadata;
 
 import java.io.IOException;
-import java.io.UncheckedIOException;
 import java.nio.channels.Channels;
 import java.nio.channels.ReadableByteChannel;
 import java.nio.channels.WritableByteChannel;
 import java.util.Collection;
-import java.util.List;
 
 import static sleeper.core.properties.instance.CdkDefinedInstanceProperty.DATA_BUCKET;
 import static sleeper.core.properties.table.TableProperty.FILES_SNAPSHOT_BATCH_SIZE;
 import static sleeper.core.properties.table.TableProperty.PARTITIONS_SNAPSHOT_BATCH_SIZE;
 
 /**
- * Saves and loads the state of a Sleeper table in Arrow files.
+ * Saves and deletes the state of a Sleeper table in Arrow files.
  */
-public class StateStoreArrowFileStore {
-    public static final Logger LOGGER = LoggerFactory.getLogger(StateStoreArrowFileStore.class);
+public class StateStoreArrowUploadFileStore {
```

Comment:
I read this as a store that holds arrow files for the state store. Should "upload" be after "file" rather than splitting up "arrow file"?